sklearn/svm/classes.py

```diff
@@ -487,7 +487,9 @@ class SVC(BaseSVC):
 
     probability : boolean, optional (default=False)
         Whether to enable probability estimates. This must be enabled prior
-        to calling `fit`, and will slow down that method.
+        to calling `fit`, and will slow down that method. When enabled,
```

Comment:
maybe "and will slow down fitting as it internally uses 5-fold cross-validation" ?

Reply:
lol now you went back to not mentioning CV (here, but mention it below ...)